xml/net_sync.xml

```diff
@@ -80,782 +74,369 @@
-        </para>
-       </entry>
-       <entry>
-        <para>
-         -
-        </para>
-       </entry>
-      </row>
-      <row>
-       <entry>
-        <para>
-         Hard Disk Space
-        </para>
-       </entry>
-       <entry>
-        <para>
-         --
-        </para>
-       </entry>
-       <entry>
-        <para>
-         o
-        </para>
-       </entry>
-      </row>
-      <row>
-       <entry>
-        <para>
-         GUI
-        </para>
-       </entry>
-       <entry>
-        <para>
-         o
-        </para>
-       </entry>
-       <entry>
-        <para>
-         -
-        </para>
-       </entry>
-      </row>
-      <row>
-       <entry>
-        <para>
-         Difficulty
-        </para>
-       </entry>
-       <entry>
-        <para>
-         o
-        </para>
-       </entry>
-       <entry>
-        <para>
-         +
-        </para>
-       </entry>
-      </row>
-      <row>
-       <entry>
-        <para>
-         Attacks
-        </para>
-       </entry>
-       <entry>
-        <para>
-         + (SSH)
-        </para>
-       </entry>
-       <entry>
-        <para>
-         +(SSH)
-        </para>
-       </entry>
-      </row>
-      <row>
-       <entry>
-        <para>
-         Data Loss
-        </para>
-       </entry>
-       <entry>
-        <para>
-         ++
-        </para>
-       </entry>
-       <entry>
-        <para>
-         +
-        </para>
-       </entry>
-      </row>
-     </tbody>
-    </tgroup>
-   </table>
-  </sect2>
- </sect1>
- <sect1 xml:id="sec.net.sync.cvs">
-  <title>Introduction to CVS</title><indexterm xml:id="idx.CVS" class="startofrange">
-
-  <primary>CVS</primary></indexterm><indexterm xml:id="idx.files_synchronizing_CVS" class="startofrange">
-
-  <primary>files</primary>
-
-  <secondary>synchronizing</secondary>
-
-  <tertiary>CVS</tertiary></indexterm>
-
-  <para>
-   CVS is suitable for synchronization purposes if individual files are edited
-   frequently and are stored in a file format, such as ASCII text or program
-   source text. The use of CVS for synchronizing data in other formats (such as
-   JPEG files) is possible, but leads to large amounts of data, because all
-   variants of a file are stored permanently on the CVS server. In such cases,
-   most of the capabilities of CVS cannot be used. The use of CVS for
-   synchronizing files is only possible if all workstations can access the same
-   server.
+   Rsync is useful when large amounts of data need to be transmitted regularly
+   while not changing too much. This is, for example, often the case when
+   creating backups. Another application concerns staging servers. These are
+   servers that store complete directory trees of Web servers that are
+   regularly mirrored onto a Web server in a DMZ.
   </para>
+  <para> Although its name suggest that Rsync is a synchronization tool, it is
+   not. By design, Rsync is a <emphasis>unidirectional</emphasis> tool, it
```

Comment:
By design, Rsync is a <emphasis>unidirectional</emphasis> tool, it copies files or directories only one way: from source to destination.

**Rsync is a tool that copies data only in one direction at a time.**